src/plugins/saved_objects_management/public/management_section/object_view/components/header.tsx

```diff
@@ -69,13 +70,16 @@ export const Header = ({
               />
             </h1>
           ) : (
-            <h1>
-              <FormattedMessage
-                id="savedObjectsManagement.view.viewItemTitle"
-                defaultMessage="View {title}"
-                values={{ title: type }}
-              />
-            </h1>
+            // </EuiTitle>
+            <EuiTitle size="l">
+              <h1>
```

Comment:
Do you have a screenshot of this one? This seems fine, but not sure

EuiText size='s'

Reply:
same as above, will replace with euitext